macros/models/_sector/dex/airswap_compatible_trades.sql

```diff
@@ -0,0 +1,53 @@
+{% macro airswap_compatible_trades(
+    blockchain = null,
+    project = null,
+    sources = null
```

Comment:
Maybe we can make default sources as empty list to hint users.
```suggestion
    sources = []
```

Reply:
good idea!